File: 2011/J113.cpp
#include <cstdio>
int main() {
  int n;
  int twos = 0;
  scanf("%d", &n);
  while (n >= 4) {
    twos++;
    n /= 2;
  }
  if (n == 3) {
    printf("3");
  }
  if (n == 2) {
    twos++;
  }
  for (int i = 0; i < twos; i++) {
    printf("2");
  }
  printf("\n");
  return 0;
}

File: 2011/J113.pas
var
  n, twos, i: longint;
begin
  read(n);
  twos := 0;
  while (n >= 4) do
  begin
    inc(twos);
    n := n div 2
  end;
  if (n = 3) then
    write(3);
  if (n = 2) then
    inc(twos);
  for i := 1 to twos do
    write(2);
  writeln
end.
